jenkins/tippecanoe_tile_join_Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+  agent {
+        node {
+            label 'node:slave'
+        }
+    }
+  stages {
+    stage('Checkout repo and pull from S3') {
+      agent any 
+      steps {
+        sh 'wget -O DOIRootCA2.cer http://sslhelp.doi.net/docs/DOIRootCA2.cer'
+        git "https://github.com/usgs-makerspace/wbeep-processing"
+        sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_data . --exclude "*" --include "hru_data.csv"'
```

Comment:
What file name should we use for the .csv?  I called it `model_output_categorized.csv`

Reply:
ok, i'll use that then. and it looks like you just have it at the root of the tier folder?